ONTOLOGY_GUIDE.md

```diff
@@ -273,65 +273,49 @@ Because we believe that it is important to ensure at least some minimal level of
+1. **Ontology imports** - No ontology may import itself. See [testHygiene1177.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene1177.sparql)
 1. **Special Characters** - The set of characters allowed in literals is limited to the alphanumeric characters, punctuation marks, and diacritic characters used in the languages FIBO uses. See [testHygiene0114.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene0114.sparql)
 1. **Unique Labels** - No label can name more than one owl:Class or rdf:Property. See [testHygiene1067.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene1067.sparql)
 1. **Circular Definitions** - No FIBO definition should be circular, i.e., a definition cannot contain a label (including synonyms) of the resource being defined. See [testHygiene1068.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene1068.sparql)
 1. **Object Property Inverses** - No property may have more than one inverse as this may, among other things, make the inverse properties equivalent. See [testHygiene1078.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene1078.sparql)
 1. **Use of rdfs:comment** - rdfs:comment shouldn't be used for FIBO annotations. See [testHygiene1079.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene1079.sparql)
 1. **Reference to owl:Thing** - We should not make explicit references to owl:Thing as these are redundant except for someValuesFrom restrictions . See [testHygiene0268.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene0268.sparql)
 1. **Synonyms as classes** - Use of OWL equivalences may indicate that different synonyms are modeled as separate classes, which should be avoided. See [testHygiene1103.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene1103.sparql)
+1. **Avoiding cycles** - Class and property hierarchies should not contain cycles. See [testHygiene1190.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene1190.sparql) and [testHygiene1190_properties.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene1190_properties.sparql)
+1. **Implicit use of “is” relationships** - Various kinds of “is” relationship should be represented by means of the appropriate logical constructs and not by means of ad-hoc properties. See [testHygiene1289.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene1289.sparql)
+1. **Disconnected resources** - All classes should be rooted in FND, LCC or FBC domain. See [testHygiene1290.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene1290.sparql)
```

Comment:
I don't agree with having this as a rule: FND (or FBC) were never intended IMO as a comprehensive set of upper level ontologies covering the universe.
Conversely we tend to have meaningless or overused properties such as "has".

Reply:
@rivettp Irrespective of the question whether your claim is justified or not this pull request does not directly concern hygiene tests, but the ontology guide, which should correspond to the current set of tests we run.
Hygiene test 1290 was added as the resolution of https://github.com/edmcouncil/fibo/issues/1290 - note that there was some discussion there on its scope and value.